chainer/functions/activation/lstm.py

```diff
@@ -136,11 +142,15 @@ def forward(self, inputs):
 
         a, i, f, o = _extract_gates(x)
         if xp is numpy:
-            tanh_a = xp.tanh(a)
-            sig_i = _sigmoid(i)
-            sig_f = _sigmoid(f)
-            sig_o = _sigmoid(o)
-
+            if intel64.should_use_ideep('>=auto'):
+                xp = intel64.ideep.get_array_module(x)
+            # TODO(iDeep) replace numpy with xp
+            tanh_a = numpy.tanh(a)
```

Comment:
Please use `xp.tanh`.

Reply:
Ah, it seems old TODO when ideep did not support it.
Fixed.